fairing/kubernetes/manager.py

```diff
@@ -101,6 +101,10 @@ def create_secret(self, namespace, secret):
         api_instance = client.CoreV1Api()
         return api_instance.create_namespaced_secret(namespace, secret)
 
+    def create_docker_secret(self, namespace, docker_secret):
```

Comment:
I remember there is same function named `create_secret`?

Reply:
Yes, thanks for pointing that out. Using existing create_secret function now.